presto-spi/src/main/java/com/facebook/presto/spi/plan/TableScanNode.java

```diff
@@ -73,6 +75,18 @@ public TableScanNode(
             Map<VariableReferenceExpression, ColumnHandle> assignments,
             TupleDomain<ColumnHandle> currentConstraint,
             TupleDomain<ColumnHandle> enforcedConstraint)
+    {
+        this (id, table, outputVariables, assignments, currentConstraint, enforcedConstraint, emptyList());
+    }
+
+    public TableScanNode(
+            PlanNodeId id,
+            TableHandle table,
+            List<VariableReferenceExpression> outputVariables,
+            Map<VariableReferenceExpression, ColumnHandle> assignments,
+            TupleDomain<ColumnHandle> currentConstraint,
+            TupleDomain<ColumnHandle> enforcedConstraint,
+            List<TableConstraint<ColumnHandle>> tableConstraints)
```

Comment:
Move this to be before the tuple domains.

Reply:
Done.